02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,51 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>The Kira Website</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
-<h1>Kira</h1>
+<body
+<div>
+  <h1>Kira Villarreal</h1>
+</div>
+  <br>
+
+<h3>About Me</h3>
+<p> Hi! I'm Kira and this is my website. I'm a generally
+  awesome person with lots of interesting talents and
+  skills. Unfortunately, coding is not one of them...yet.
+  But stay tuned. In the meantime, here's some awesome
+  stuff about me.
+</P>
+<h3>My Favorite Things</h3>
+<ul>
+  <li>Tennis</li>
+  <li>Singing</li>
+  <li>Wine</li>
+  <li>Travel</li>
+  <li>Dogs</li>
+  <li>Baseball</li>
+</ul>
+<h3>Favorite Websites</h3>
+<ul>
+  <li>I'm a huge fan of the Baltimore Orioles, so I like <tbody>
+    check their scores and highlights daily. You can find the Orioles
+    website <a href="http://baltimore.orioles.mlb.com/index.jsp?c_id=bal">
+    here.</a> Also, the Yankees suck!
+  </li>
+  <li>Here's a link to the facebook page for my acoustic duo,
+    <a href="https://www.facebook.com/bubblesandbeersa/?ref=aymt_homepage_panel">
+    Bubbles and Beer. </a> We'll have a real website someday. Once I learn enough
+    to build it.
+  </li>
+  <li>And <a href="https://www.amazon.com/">this one </a>. Because I like to buy all the things.</li>
```

Comment:
adding the attribute `target="_blank"`  to your <a> tags will make the link open in a new window — that way people don't lose their page with your site! Not ever _necessary_, but often times nice to have.